### PR TITLE
Add Spots

### PR DESCRIPTION
**Pull request by zenangst**
Spots is a view controller framework that makes your setup and future development blazingly fast.

### CODE REVIEW

**Comment by zenangst** [comment]
Bump! 😁

**Comment by vasiliyb** [comment]
dont think anyone is paying attention :(  g2g otherwise :)

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ A curated list of awesome Swift frameworks, libraries and software. Inspired by
 * [SAHistoryNavigationViewController](https://github.com/szk-atmosphere/SAHistoryNavigationViewController) - SAHistoryNavigationViewController realizes iOS task manager like UI in UINavigationContoller.
 * [WobbleView](https://github.com/inFullMobile/WobbleView) - Implementation of wobble effect for any view in app.
 * [SimpleAlert](https://github.com/KyoheiG3/SimpleAlert) - Customizable simple Alert and simple ActionSheet for Swift
+* [Spots](https://github.com/hyperoslo) - Spots is a view controller framework that makes your setup and future development blazingly fast.
```
Comment (maxerogers) [inline README.md]: should be `https://github.com/hyperoslo/Spots` not `https://github.com/hyperoslo`

Comment (zenangst) [inline README.md]: Fixed 😁

**Comment by zenangst** [comment]
Bump!

**Pull request closed without merging**